src/styles/main.scss

```diff
@@ -1 +1,60 @@
 // put code here
+@import "./variables";
+
+.display {
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  box-sizing: border-box;
+  width: $box-size;
+  height: $box-size;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border: $border-size dotted;
+  border-radius: 50%;
+}
+
+.display_center {
+  width: $center-point-size;
+  height: $center-point-size;
+  border-radius: 50%;
+  background-color: red;
+  z-index: 1;
+}
+
+.display_seconds-hand {
```

Comment:
The class naming does not follow BEM (Block Element Modifier) methodology. The class should be named as 'display__seconds-hand' instead of 'display_seconds-hand'.